tests/ci/docker_images/linux-ppc64le/ubuntu-build-tools/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+FROM ppc64le/ubuntu:latest
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update && apt-get install -y  \
+    gcc g++ cmake golang gdb valgrind \
+    libclang-dev clang \
+    build-essential \
+    ssh \
+    rsync \
+    tar \
+    python3 \
+	&& apt-get clean \
+    && mkdir -p /.cache \
+    && chmod 777 /.cache
```

Comment:
Do we want to add those lines to the other images with a comment that they were needed for running in some IDE?

Reply:
I plan to set the GOCACHE environment variable, so this wouldn't be needed.